src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -33,13 +35,35 @@ impl KbcInterface for Kbc {
         Err(anyhow!("Decrypt Payload API of this KBC is unimplemented."))
     }
 
-    async fn get_resource(&mut self, _description: &str) -> Result<Vec<u8>> {
-        Err(anyhow!("Unimplemented"))
+    #[allow(unused_assignments)]
+    async fn get_resource(&mut self, description: &str) -> Result<Vec<u8>> {
+        let desc: ResourceDescription = serde_json::from_str::<ResourceDescription>(description)?;
+        let mut resource_url = String::default();
+        let resource_type = desc
+            .optional
+            .get("type")
+            .ok_or_else(|| anyhow!("Invalid Resource description: Missing `type` filed"))?;
+        let resource_tag = desc
+            .optional
+            .get("tag")
+            .ok_or_else(|| anyhow!("Invalid Resource description: Missing `tag` filed"))?;
+        if let Some(resource_repo) = desc.optional.get("repository") {
```

Comment:
Seems that the kbs' url in `ResourceDescription` will overwrite the url of the `cc-kbc` here. Is the reverse more appropriate?